pkg/actions/event.go

```diff
@@ -37,6 +38,7 @@ func marshalEventInformation(actionName, hookID string, record graveler.HookReco
 		CommitMessage:  record.Commit.Message,
 		Committer:      record.Commit.Committer,
 		CommitMetadata: record.Commit.Metadata,
+		MergeSource:    record.MergeSource.String(),
```

Comment:
Is this left out from lua type on purpose? (https://github.com/treeverse/lakeFS/blob/master/pkg/actions/lua.go#L47)

Reply:
No, I is a bug - will fix it